applications/crossbar/src/modules/cb_system_configs.erl

```diff
@@ -293,3 +308,16 @@ get_system_config(Config, Node) ->
     System = maybe_new(kapps_config:get_category(Config)),
     SystemNode = kz_json:get_value(Node, System, kz_json:new()),
     kz_json:merge_recursive(Default, SystemNode).
+
+-spec make_schema(ne_binary()) -> kz_json:object().
+make_schema(Id) ->
+    Flat = [
+        {[<<"patternProperties">>, <<".+">>, <<"$ref">>], kapps_config_util:system_schema_name(Id)}
+        ,{[<<"patternProperties">>, <<".+">>, <<"type">>], <<"object">>}
+        ,{<<"type">>, <<"object">>}
+    ],
```

Comment:
can we change this to
```
    Flat = [
            {<<"$schema">>,<<"http://json-schema.org/draft-04/schema#">>}
           ,{<<"id">>, <<"system_config">>}
           ,{<<"type">>, <<"object">>}
           ,{<<"default">>, kz_json:new()}
           ,{[<<"properties">>, <<"default">>, <<"$ref">>], kapps_config_util:system_schema_name(Id)}
           ,{[<<"properties">>, <<"default">>, <<"default">>], kz_json:new()}
           ,{[<<"properties">>, <<"default">>, <<"type">>], <<"object">>}
           ,{[<<"patternProperties">>, <<".+">>, <<"$ref">>], kapps_config_util:system_schema_name(Id)}
           ,{[<<"patternProperties">>, <<".+">>, <<"type">>], <<"object">>}
           ],
```
it will ensure the `default` is present and with defaults